### ISSUE TITLE
Ubuntu 16 EOL.  Ubuntu18 support?

### ISSUE THREAD

**Issue by winterz** [probonopd/linuxdeployqt#481]
Ubuntu Linux 16.04 LTS will no longer be supported by April 30, 2021. 

I wonder if there are any plans to start supporting Ubuntu Linux 18 LTS

**Comment by probonopd** [comment]
Yes, after April 30, 2021 we will start looking into allowing building AppImages on Ubuntu LTS (Bionic Beaver). Although there is no need to - applications built on 16.04 LTS should continue to run on subsequent versions just fine.

**Comment by winterz** [comment]
@probonopd Thanks for the super-fast response.  And happy to hear it.

**Comment by TheAssassin** [comment]
> Although there is no need to - applications built on 16.04 LTS should continue to run on subsequent versions just fine.

That is just half of the story. Of course it will *function*. But the nonfunctional security requirements cannot be provided any more by EOL distributions. So keep using the oldest still-supported LTS. IMO, just upgrade already to 18.04. Nobody will, like, upgrade from xenial in exactly 2 weeks. If they haven't yet, they probably won't anyway.

**Comment by winterz** [comment]
The problem is that our admins want to upgrade to upgrade our CI worker to 18.04 ASAP (rightly so). 
once that happens we cannot create an appimage for a new version of our software until linuxdeployqt properly supports 18.04

**Comment by TheAssassin** [comment]
IMO this paternalizing behavior in this project ignores real world needs. I don't like to advertise own software normally in other people's issue trackers, but you may want to check out linuxdeploy, which follows a different approach. I would, if any, at most warn users about not using the oldest still-supported LTS release.

**Comment by winterz** [comment]
@probonopd if I sounded angry, mad or otherwise unfriendly that was not my intention.  I have no complaints with linuxdeployqt and am only hoping that sometime in the not-so-distant future there is a version for ubuntu18.   This is a wonderful tool and we very much appreciate the efforts of everyone involved.
I apologize if I conveyed negative vibes.
-Allen

**Comment by Milerius** [comment]
This will also a big problem for our product of we are not able to build appimage on the GitHub ci drop 16.04

**Comment by alemuntoni** [comment]
Same problem here.
Ubuntu 16.04 is going to be retired, as they state here: https://github.com/actions/virtual-environments/issues/3082
Without deployqt working on ubuntu 18.04, we won't be able to build and release our software for linux.

**Comment by probonopd** [comment]
Onoce Ubuntu 16.04 is retired, the restriction will be relaxed.

**Comment by ouuan** [comment]
[GitHub Actions : Ubuntu 16.04 LTS virtual environment will be removed on September 20, 2021](https://github.blog/changelog/2021-04-29-github-actions-ubuntu-16-04-lts-virtual-environment-will-be-removed-on-september-20-2021/)

**Comment by winterz** [comment]
so far so good.
Thank you !
